modules/apps/archived/archived-modules-upgrade/src/main/java/com/liferay/archived/modules/upgrade/internal/UpgradePortalDirectory.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.archived.modules.upgrade.internal;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Sam Ziemer
+ */
+public class UpgradePortalDirectory extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		LayoutTypeSettingsUtil.removePortletId(connection, "11");
```

Comment:
@SamZiemer for this case it is the same but we have to take into account that this plugin handles 4 portlets so we will need to verify 8 portletIds. See:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/archived/directory-web/src/main/java/com/liferay/directory/web/internal/upgrade/v1_0_0/UpgradePortletId.java#L28